binding/lua/multiverso-scm-1.rockspec

```diff
@@ -0,0 +1,35 @@
+package = "multiverso"
+version = "scm-1"
+
+source = {
+    url = "https://github.com/Microsoft/multiverso"
+}
+
+description = {
+    summary = "Torch binding for multiverso.",
+    detailed = [[
+        Multiverso is a parameter server framework for distributed machine
+        learning, this package make it possible to have multiverso embedded in
+        your original torch programs.
```

Comment:
Multiverso is a parameter server framework for distributed machine learning. this package can leverage multiple machines and GPU to speed-up the torch programs.